src/Symfony/Component/HttpKernel/Attribute/Reset.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Attribute;
+
+/**
+ * @author Alexander M. Turek <me@derrabus.de>
+ */
+#[\Attribute(\Attribute::TARGET_CLASS | \Attribute::IS_REPEATABLE)]
+final class Reset
```

Comment:
I think we should not add an attribute when an autoconfiguration rule already exists for the same feature. Here, I would encourage ppl to implement `ResetInterface` instead of adding the attribute.

If you don't agree, please still remove the attribute from the PR, to not make this a blocker.

Reply:
Makes sense.